src/main/java/com/conveyal/datatools/manager/controllers/api/ProjectController.java

```diff
@@ -326,12 +327,7 @@ public static ScheduledFuture scheduleAutoFeedFetch (Project project, int interv
             // First cancel any already scheduled auto fetch task for this project id.
             cancelAutoFetch(project.id);
 
-            ZoneId timezone;
-            try {
-                timezone = ZoneId.of(project.defaultTimeZone);
-            }catch(Exception e){
-                timezone = ZoneId.of("America/New_York");
-            }
+            ZoneId timezone = getTimezone(project.defaultTimeZone);
```

Comment:
Perhaps this `scheduleAutoFeedFetch` and the `FeedSource#scheduleExpirationNotifications` (and the respective cancel methods) should all go in the same class. It makes me uncomfortable that this logic is getting spread out and I suspect there is some duplicated code that could be factored out. Maybe this could all go in the `datatools.common.util` package as `ScheduleUtils`? Some of the logic in `DataManager.java` might belong here also.

Reply:
I suppose that make sense.  I'll try to do that.

A few other things I saw: it looks like the logic to schedule auto-fetch of specific feed sources is missing in datatools-server.  Also, it looks like we need to reconfigure project creation to automatically schedule fetches for a project upon creation in light of the changes in https://github.com/catalogueglobal/datatools-ui/pull/41

Reply:
Wait nvm, I see that project auto-fetch must be enabled for a feed to automatically be fetched.